.envrc

```diff
@@ -104,8 +104,10 @@ export USE_TLS=false
 # Add local paths for binaries and scripts
 PATH_add ./bin
 
-export CEDAR_ENV=dev
-export CEDAR_API_URL="webmethods-apigw.cedardev.cms.gov"
+# CEDAR
+export CEDAR_CORE_MOCK=true
+export CEDAR_INTAKE_DISABLED=true
```

Comment:
1 vote for changing this to `CEDAR_INTAKE_ENABLED` to prevent us from mentally converting `DISABLED=false` into `ENABLED` constantly when reading the code.

Reply:
We made it `DISABLED` so that the default behavior in hosted envs was to be on without having to change any env vars, and it made both flags "overrwide to disable", rather than "set to enable"

Maybe we can discuss in a review, once this is out of draft?

Reply:
@mynar7 I updated this here!

https://github.com/CMSgov/easi-app/pull/2487/commits/d6fa0d847a261ae7e8a9644640a3294d1c3160a8